manifests/init.pp

```diff
@@ -38,11 +42,12 @@
   Hash              $crons,
   Array[String]     $dependencies,
   Integer           $logs_archive_after_days,
-  Optional[String]  $logs_archive_dir,
   Integer           $logs_compress_after_days,
   String            $logs_mgmt_script_path,
   String            $logs_path,
   Array[String]     $required_pkgs,
+  Hash              $custom_files = {},
```

Comment:
These 2 lines look odd to what I normally expect.

I've generally only assigned a default value here if a parameter is marked `Optional`. 

- Here the Optional parameter is not getting assigned a default value
- And the non-optional parameter is getting assigned a default

Perhaps that works, but is backwards to how I generally do it.